### ISSUE TITLE
all files in legal directory should not be group or world executable and writable

### ISSUE THREAD

**Issue by fawaf** [corretto/corretto-11#129]
### Describe the bug
all files in legal directory are currently owner, group, and world readable, executable, and writable.

### To Reproduce
1. login to a host with the java-11-amazon-corretto-devel package installed
1. `cd /usr/lib/jvm/java-11-amazon-corretto/legal`
1. `ls` the directories in said directory

### Expected behavior
all files in legal directory should be readable by everyone, but only writable by the owner. fixing this also allows passing of CIS benchmark "6.1.10 Ensure no world writable files exist"

### Screenshots
`ls` output from an affected host:
```
$ ls -l /usr/lib/jvm/java-11-amazon-corretto/legal/*
...snip...
jdk.xml.dom:
total 28
-rwxrwxrwx 1 root root  2114 Jul 11  2019 ADDITIONAL_LICENSE_INFO
-rwxrwxrwx 1 root root  1522 Jul 11  2019 ASSEMBLY_EXCEPTION
-rwxrwxrwx 1 root root 19274 Jul 11  2019 LICENSE

jdk.zipfs:
total 28
-rwxrwxrwx 1 root root  2114 Jul 11  2019 ADDITIONAL_LICENSE_INFO
-rwxrwxrwx 1 root root  1522 Jul 11  2019 ASSEMBLY_EXCEPTION
-rwxrwxrwx 1 root root 19274 Jul 11  2019 LICENSE
09/03 02:46[legal]$ 
```

### Platform information
    OS: Amazon Linux 2
    Version:
```
openjdk version "11.0.4" 2019-07-16 LTS
OpenJDK Runtime Environment Corretto-11.0.4.11.1 (build 11.0.4+11-LTS)
OpenJDK 64-Bit Server VM Corretto-11.0.4.11.1 (build 11.0.4+11-LTS, mixed mode)
```

### Additional context
Add any other context about the problem

**Comment by cliveverghese** [comment]
Hi @fawaf 

Thank you for reporting this issue, We are looking into it and will provide you an update soon.

**Comment by lutkerd** [comment]
This is resolved by #130
